cli/command/environment_test.go

```diff
@@ -0,0 +1,312 @@
+package command
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/quintilesims/layer0/api/job"
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/urfave/cli"
+)
+
+func TestEnvironmentCommand_userInputErrors(t *testing.T) {
+	base, ctrl := newTestCommand(t)
+	defer ctrl.Finish()
+
+	command := NewEnvironmentCommand(base.Command())
+
+	testCases := []struct {
+		name    string
+		command func(*cli.Context) error
+		args    []string
+	}{
+		{
+			name:    "create",
+			command: command.create,
+		},
+		{
+			name:    "update",
+			command: command.update,
+		},
+		{
+			name:    "setMinCount",
```

Comment:
This is confusing - why are setMinCount and update treated as separate commands?